src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -13,6 +16,13 @@ import scala.util.Try
 
 class LocalBackend extends Backend {
 
+  implicit class WriteWithNewline(writer: Writer) {
```

Comment:
Because I wasn't using it outside of here, it could be moved to an object if other code can use it.